internal/operands/metrics/resources.go

```diff
@@ -73,100 +78,104 @@ var RecordRulesDescList = []RecordRulesDesc{
 	},
 }
 
-var alertRulesList = []promv1.Rule{
-	{
-		Expr:   intstr.FromString("sum(kubevirt_vmi_phase_count{phase=\"running\"}) by (node,os,workload,flavor)"),
-		Record: "cnv:vmi_status_running:count",
-	},
-	{
-		Alert: "SSPDown",
-		Expr:  intstr.FromString("kubevirt_ssp_operator_up_total == 0"),
-		For:   "5m",
-		Annotations: map[string]string{
-			"summary":     "All SSP operator pods are down.",
-			"runbook_url": runbookURLBasePath + "SSPOperatorDown",
-		},
-		Labels: map[string]string{
-			severityAlertLabelKey:     "critical",
-			healthImpactAlertLabelKey: "critical",
-			partOfAlertLabelKey:       partOfAlertLabelValue,
-			componentAlertLabelKey:    componentAlertLabelValue,
-		},
-	},
-	{
-		Alert: "SSPTemplateValidatorDown",
-		Expr:  intstr.FromString("kubevirt_ssp_template_validator_up_total == 0"),
-		For:   "5m",
-		Annotations: map[string]string{
-			"summary":     "All Template Validator pods are down.",
-			"runbook_url": runbookURLBasePath + "SSPTemplateValidatorDown",
-		},
-		Labels: map[string]string{
-			severityAlertLabelKey:     "critical",
-			healthImpactAlertLabelKey: "critical",
-			partOfAlertLabelKey:       partOfAlertLabelValue,
-			componentAlertLabelKey:    componentAlertLabelValue,
-		},
-	},
-	{
-		Alert: "SSPFailingToReconcile",
-		Expr:  intstr.FromString("(kubevirt_ssp_num_of_operator_reconciling_properly == 0) and (kubevirt_ssp_operator_up_total > 0)"),
-		For:   "5m",
-		Annotations: map[string]string{
-			"summary":     "The ssp-operator pod is up but failing to reconcile",
-			"runbook_url": runbookURLBasePath + "SSPFailingToReconcile",
-		},
-		Labels: map[string]string{
-			severityAlertLabelKey:     "critical",
-			healthImpactAlertLabelKey: "critical",
-			partOfAlertLabelKey:       partOfAlertLabelValue,
-			componentAlertLabelKey:    componentAlertLabelValue,
+func getAlertRules() []promv1.Rule {
+	runbookURLTemplate, _ := getRunbookURLTemplate()
```

Comment:
When returning an error of course you should also check it and possibly pass it up in the call stack. Right now the possible error from `getRunbookURLTemplate` is just ignored.

Reply:
Updated, please let me know if the change makes sense